scrapers/privatecastings.yml

```
@@ -0,0 +1,21 @@
+name: privatecastings
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - privatecastings.com/scene/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h1
+      Performers:
+        Name: //ul[@class="scene-models-list"]//a
+      Details: //section/h1[1]
+      Tags:
+        Name: //ul[@class="scene-tags"]//li
+      Image: //meta[@itemprop="thumbnailUrl"]//@content
```

typo ( content is a meta attribute so `//` isnt needed )
```suggestion
      Image: //meta[@itemprop="thumbnailUrl"]/@content
```